internal/plugin/deleteplugin.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018, 2019 the Velero contributors.
```

Comment:
Current project coding standard is to omit the year here. Just "Copyright the Velero contributors."

Reply:
Fixed. Thanks.